app/facades/foods_facade.rb

```diff
@@ -0,0 +1,16 @@
+class FoodsFacade
+
+  class << self
+
+    def get_foods(food)
+      json = FoodsService.find_food(food)[:foods]
+
+      json[0..9].map {|food| Food.new(food)}
```

Comment:
You can also limit the # of responses back from the API using the `pageSize` query parameter it provides. Another option to avoid having to limit it with Ruby here!